exercises/concept/poetry-club-door-policy/.docs/instructions.md

```diff
@@ -6,7 +6,7 @@ Because there have been incidents in the past, the club has a very specific door
 There are two doors at the poetry club, both are guarded.
 In order to gain entry, you'll need to work out the password of that day:
 
-## Front door
+**Front door**
```

Comment:
It sucks but this can't be a heading... any h2 will be interpreted as a task:

<img width="835" alt="Screen Shot 2021-07-24 at 18 26 35" src="https://user-images.githubusercontent.com/7852553/126875651-df44722f-7131-43b2-ad73-692a56e641f3.png">

Reply:
Ah yes. See, it's THAT old ;)